README.md

```diff
@@ -49,23 +49,31 @@ What are the types of the following expressions and what do they evaluate to?
 
 ```js
 17
+// 17
```

Comment:
Remember to always to read and do the whole question. You're missing saying the type.

Reply:
We noticed right after submitting. Will be more careful with the Variables lab and future labs. Thank you.